creator-node/src/snapbackSM/secondarySyncHealthTracker.js

```diff
@@ -169,17 +169,42 @@ const Getters = {
       logger.error(`SecondarySyncHealthTracker - getSyncRequestOutcomeMetrics() Error || ${e.message}`)
       return {}
     }
+  },
+
+  /**
+   * Returns single int representing SyncRequestOutcome for secondary, wallet, syncType, date=today, and Outcome=Failure
+   * Only one redis key should exist for above params, but takes 1st value if multiple are found
+   */
+  async getSecondaryUserSyncFailureCountForToday (secondary, wallet, syncType) {
```

Comment:
i feel like we can generalize this, but not as egregious as the other redis tracking file. not a blocker for now

Reply:
well it is easily generalizable but i added this specifically as a convenience function so the caller doesn't need to parse the response etc

Reply:
yes i agree. not an action item for this now